back/engines/commercial/content_builder/spec/acceptance/layout_images_spec.rb

```diff
@@ -43,14 +43,6 @@
         )
       end
 
-      example '[error] Create a layout image without an image', document: false do
-        do_request(layout_image: { image: '' })
-
-        assert_status 422
-        json_response = json_parse response_body
-        expect(json_response).to include_response_error(:image, 'blank')
-      end
-
```

Comment:
If my suggested solution in #3902, to the validation issues related to template application is accepted, then we will be able to replace this test.